apple-nav/src/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import NavWrapper from "./navigation/NavWrapper";
+import SubNav from "./navigation/SubNav";
+import { Route } from "react-router-dom";
+
```

Comment:
good job on keeping your app.js clean and importing everything. This is how an app.js should usually look